docs/faker/realtime.md

```diff
@@ -0,0 +1,29 @@
+## Realtime fake generator
```

Comment:
If we have cli options for realtime fake generator add an example. Maybe is int the cli documentation.